font/src/ft/fc.rs

```diff
@@ -11,42 +11,34 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
-use std::fmt;
-use std::ptr;
-
+use std::{fmt, ptr};
+
+use fontconfig::fontconfig::{
```

Comment:
Having a `fontconfig::fontconfig` import is very confusing, which is why the original version had `use fontconfig::fontconfig as ffi` as an easier to understand import. I think that makes a lot of sense and makes it a ton easier to understand what exactly we're importing here. So I'd be in favor of changing that back.

Reply:
I was actually tempted to rebind it to the name of the crate it came from. I'll add ffi back in and bind off of that.